test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -93,7 +91,11 @@ trait CollectionTest[F <: CollectionFactory] extends JasmineTest {
     }
 
     it("should store custom objects") {
-      case class TestObj(num: Int)
+      case class TestObj(num: Int) extends jl.Comparable[TestObj] {
+        def compareTo(o: TestObj): Int =
+          if (o==null) -1
```

Comment:
But I guess these collections do not accept `null` elements, in that case? At least not with the default comparator. I wouldn't make sense in the other direction, if you try to do `null.compareTo(obj)`. And it is generally not a requirement of `compareTo` to behave well when given `null` (unlike `equals`). I would blame the collection for giving `null` to `compareTo` in the first place.